src/contracts/middleware/VoteWeigherBase.sol

```diff
@@ -100,25 +110,16 @@ contract VoteWeigherBase is VoteWeigherBaseStorage {
      */
     function removeStrategiesConsideredAndMultipliers(
         uint8 quorumNumber,
-        IStrategy[] calldata _strategiesToRemove,
         uint256[] calldata indicesToRemove
-    ) external virtual onlyServiceManagerOwner {
-        uint256 numStrats = _strategiesToRemove.length;
-        // sanity check on input lengths
-        require(indicesToRemove.length == numStrats, "VoteWeigherBase.removeStrategiesConsideredAndWeights: input length mismatch");
-
-        for (uint256 i = 0; i < numStrats;) {
-            // check that the provided index is correct
-            require(
-                strategiesConsideredAndMultipliers[quorumNumber][indicesToRemove[i]].strategy == _strategiesToRemove[i],
-                "VoteWeigherBase.removeStrategiesConsideredAndWeights: index incorrect"
-            );
-
+    ) external virtual onlyServiceManagerOwner validQuorumNumber(quorumNumber) {
+        uint256 indicesToRemoveLength = indicesToRemove.length;
+        require(indicesToRemoveLength > 0, "VoteWeigherBase.removeStrategiesConsideredAndMultipliers: no indices to remove provided");
+        for (uint256 i = 0; i < indicesToRemoveLength;) {
+            emit StrategyRemovedFromQuorum(quorumNumber, strategiesConsideredAndMultipliers[quorumNumber][indicesToRemove[i]].strategy);
```

Comment:
we enforce elsewhere (when creating new quorums) that each quorum contains at least one strategy.
seems like this is a property we _might_ want to ensure
IMO it's better to not have this property, but if we agree it's OK then we should have a test that specifically removes all strategies from a quorum and then tests the behavior (I guess the desired behavior would be just return 0 weight for everyone and not revert?)

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/53/commits/8ee2fc0de766656c46ac2d92ae089703959d8f60 agree the functionality is not needed.